### ISSUE TITLE
De nouveaux champs?

### ISSUE THREAD

**Issue by Enimie** [maieul/biblatex-manuscripts-philology#3]
Coucou,

Je suis en train de me créer ma base de données de manuscrits sur biblatex avec ton package.
Il manque, je crois des champs qui sont nécessaires. En voici une petite liste:

- "type" : manuscrit homogène, hétérogène, ou membrum disjectum (avec membrum disjectum, il faudrait pouvoir rajouter la ref d'un ou des  autres manuscrits qui proviennent éventuellement du même manuscrit. donc plutôt un champ format texte)
- provenance (n'est pas comme localisation, ni l'origine)
- nombre de lignes par page/colonne
-réglures
-Format
-Surface ecrite
-Reliure
-Ecriture
-décoration
-Organisation (pour marquer : les cahiers, les réclames, les signatures)
-colophon ou autre indication
-contenu

Qu'en penses-tu?

Merci en tout cas!

E.

**Comment by maieul** [comment]
oui, après reste à savoir si on intègre ces champs dans les styles par défaut ou pas 

tu traduis cela en anglais ? tu peux modifier les fichier biblatex-manuscript-shared.dbx pour ajouter ces champs ?

**Comment by Enimie** [comment]
Ok, je fais ça.

Je pense qu'il faut mettre le format en champ par défaut ;  le reste, sans doute pas

**Comment by maieul** [comment]
pour format, prévoit un champ width et un champ height, ca permet d'adapter aux conventions du pays

**Comment by maieul** [comment]
évidement l'idéal serait que tu fasse toi même le commit dans une copie du dépôt=

**Comment by Enimie** [comment]
Ok pour le champ champ width et le champ height. J'ai aussi crée une nouvelle commande de citation et le driver qui va avec, pour imprimer des notices descriptives des manuscrits (mais je cherche encore comment l'appeler en anglais). Je poste tout ca quand je l'aurai mis au propre, pour l'instant c'est encore brouillon.

**Comment by maieul** [comment]
on attend les nouveau champs ;-)

**Comment by Enimie** [comment]
Désolé, j'avais oublié!

Voici pour l'instant mes nouveaux champs (ou déjà existant, comme title, mais avec une utilisation spéciale): 
ruling: les réglures
provenance:différents de origine
decor
binding:la reliure
widthl:largeur du manuscrit
height:hauteur
texth:hauteur du texte écrit ("miroir")
textw:largeur du texte ecrit
colophon
writing:pour mettre des indications sur le ou les types d'écriture (caroline, gothique, etc)
exlibris
lines:nombre de lignes par page ou par colonne
layout:tout renseignement sur la disposition de la pages
pricking:indication de piqures (je ne l'ai pas encore intégré, j'hesite sur son utilité)
type. Trois possibilités:  homogeneous (manuscrit homogene), composite (manuscrit composé de plusieurs elements), ou  disiectum (pour membrum disjectum)
typedetails:pour préciser par exemple le nombre d'unité codicologiques dans le manuscrit composite..
genre:pour donner des indications comme "bible", "florilège poétique", etc
composition:organisation des cahiers, folios inséréss, etc
gloses:pour tout ce qui est gloses ou annotations.
title:exemple "Bible de Charles le Chauve"

Voilà. J'ai aussi fait deux nouvelles commandes de citations:
-  \mcite, qui est intermédiaire entre la citation complète et le shcite. Ex: Paris, BnF, Latin 2
-\notice, pour imprimer une notice de manuscrit très détaillée. Je ne sais pas si c'est utile.

Un environnement notice: permet d'imprimer la bibliographie sous forme de suite de notices (c'est en fait ce dont j'avais besoin).

Je t'envoie ça par mail?  Ce n'est pas encore très propre, mais j'espère que tout est clair, j'ai beaucoup annoté. Je l'utilise quotidiennement en ce moment, donc c'est testé.

**Comment by maieul** [comment]
oui envoi cela par mail

**Comment by maieul** [comment]
pour genre, pourquoi ne pas prendre le champ standard subtype? c'est celui qui est utilis� par les autres types, et cela permet de faire facilement un filtre dans la biblio finale

**Comment by maieul** [comment]
des nouvelles ? j'aimerais bien intégrer notamment le champ "provenance" pour distinguer de l'origine (j'imagine provenance = lieu de passage avant acquisition par bibliothèque, origine = lieu de copie).
